css/_fonts.css

```diff
@@ -0,0 +1,4 @@
+@font-face {
+    font-family: 'Encode Sans', sans-serif;
+    src: url("../fonts"); 
```

Comment:
Seems like you're not actually pointing to the font files. Here is just a general url to the `fonts` folder.
You should have multiple `@font-face` definitions here, one for each font style you include.
When you downloaded the font from Google, you should have gotten also a css file with the needed font-faces defined, and you can just copy-paste them in this file (and check that the paths to the fonts files are correct to your folder structure).